Lib/test/_test_multiprocessing.py

```diff
@@ -4390,9 +4390,16 @@ def record(*args):
             signal.signal(signal.SIGUSR1, oldhandler)
 
 #
-# Test to verify handle verification, see issue 3321
+# Test to verify handle verification, see issue 3321 and 40402
 #
 
+class SlowPicklable:
+    def __reduce__(self):
+        from time import sleep
+        sleep(1)
```

Comment:
Please, don't use sleep() as a synchronization method:
https://pythondev.readthedocs.io/unstable_tests.html#don-t-use-sleep-as-synchronization

cc @pablogsal

Reply:
👆

Reply:
Sorry about that, it's fixed.

Reply:
No problem. It's just that in my experience, each test using sleep will fail one day.